package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "rapidpro2pg",
+  "version": "0.0.1",
+  "description": "Adapter that pulls data from RapidPro and makes it available in PostgreSQL for ease of analysis.",
```

```suggestion
  "description": "Pulls data from RapidPro and makes it available in PostgreSQL for ease of analysis.",
```

"Adapter" sounds like a library or component that sits in the middle of two components, but that doesn't have its own process (just an opinionated comment).